logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/message/ephemeral/DeleteEphemeralMessageForSelfUserAsReceiverUseCase.kt

```diff
@@ -58,23 +60,53 @@ internal class DeleteEphemeralMessageForSelfUserAsReceiverUseCaseImpl(
     private val currentClientIdProvider: CurrentClientIdProvider,
     private val messageSender: MessageSender,
     private val selfUserId: UserId,
+    private val selfConversationIdProvider: SelfConversationIdProvider
 ) : DeleteEphemeralMessageForSelfUserAsReceiverUseCase {
     override suspend fun invoke(conversationId: ConversationId, messageId: String): Either<CoreFailure, Unit> =
-        currentClientIdProvider().flatMap { currentClientId ->
-            messageRepository.getMessageById(conversationId, messageId)
-                .flatMap { message ->
-                    sendDeleteMessageToOriginalSender(
-                        message.id,
-                        message.conversationId,
-                        message.senderUserId,
-                        currentClientId
-                    ).onSuccess {
-                        deleteMessageAssetIfExists(message)
+        messageRepository.markMessageAsDeleted(messageId, conversationId).flatMap {
```

Comment:
this change is because the message content is not deleted when the user is offline or there is issues when sending the delete message